go.mod

```diff
@@ -9,6 +9,8 @@ require (
 	github.com/brianstrauch/cobra-shell v0.4.0
 	github.com/client9/gospell v0.0.0-20160306015952-90dfc71015df
 	github.com/confluentinc/ccloud-sdk-go-v1-public v0.0.0-20230427001341-5f8d2cce5ad9
+	github.com/confluentinc/ccloud-sdk-go-v2-internal/flink v0.0.13
```

Comment:
are we ok with using `internal` v2? isn't it private?

Reply:
Thanks for the reminder. The Flink team needs to add this package.